build.gradle

```diff
@@ -59,7 +59,7 @@ dependencies {
     implementation group: 'org.opengeo', name: 'geodb', version: '0.8'
     testImplementation('org.springframework.boot:spring-boot-starter-test')
     testImplementation('junit:junit:4.12')
-    testImplementation('org.mockito:mockito-core:2.1.0')
+    testImplementation('org.mockito:mockito-core:2.13.0')
```

Comment:
I had to upgrade our Mockito version because the old version we were using was missing some implementations we needed.